code/frameworks/nextjs/src/imports/webpack.ts

```diff
@@ -2,8 +2,14 @@ import TsconfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
 import { loadConfig } from 'tsconfig-paths';
 import type { Configuration as WebpackConfig } from 'webpack';
 
-export const configureImports = (baseConfig: WebpackConfig): void => {
-  const configLoadResult = loadConfig();
+export const configureImports = ({
+  baseConfig,
+  configDir,
+}: {
+  baseConfig: WebpackConfig;
+  configDir: string;
+}): void => {
+  const configLoadResult = loadConfig(configDir);
```

Comment:
do you need to handle configDir being undefined?

Reply:
no, because passing undefined to `loadConfig` would be the default case and `loadConfig` would try to resolve from `cwd` instead.